docs/wiki/archive/Magnetometer.mdx

```diff
@@ -17,56 +20,65 @@ Mag information is essential for position hold, which we intend to support.
 
 GPS Rescue does not require Mag data, but heading control may be improved if reliable, accurate Mag information is available, especially during descents on windy days. Note: DO NOT enable Mag if you rely on GPS rescue, unless you are ABSOLUTELY CERTAIN that the Mag data is accurate and reliable, and have confirmed that enabling Mag makes a significant improvement in rescue behaviour!
 
-## What is a Magenetometer?
+## What is a Magenetometer? {#magnetometer-explanation}
 
-A magnetometer is a three-axis device that detects the strength of the earth's magnetic field on each of three orthogonal sensor axes.
+A magnetometer is a three-axis device that detects the local strength and direction of the [earth's magnetic field](https://en.wikipedia.org/wiki/Earth%27s_magnetic_field)[^1].
 
-Magnetometers output data on three axes: X, Y and Z. Typically, these axes are arranged as follows:
+Magnetometers output data on three axes: X, Y and Z. The orientation of the axes can be arbitrary, depending on the magnetometer model. But typically, the axes are arranged as follows:
 
-- Y is 90 degrees to the left of X
-- Z is up
+- Y is pointing 90° to the left of X
+- Z is pointing up
 
-The reported value for the field strength on any particular axis will be positive, and maximal, when it is closely aligned with the axis of the local magnetic field, and is pointing towards the North of that field. It will be maximally negative when aligned with the South of the local magnetic field.
+The reported value for the field strength on any particular axis will be positive, and maximal, when it is aligned parallel to the orientation of the local magnetic field (pointing towards North). It will be maximally negative when aligned antiparallel to the local magnetic field (pointing towards South). The other two axes will report values close to zero. This way you can determine that one axis is perfectly aligned with the local magnetic field.
 
```

Comment:
Maybe change  `(pointing towards North)` to `(pointing towards magnetic North, at the correct inclination angle)`, and `(pointing towards South)` to `(pointing towards magnetic South, at the opposite inclination angle)`